### PR TITLE
add tutorial for release v0.2.0 to support EdgeX 2.1

### LINKED ISSUE #34: [feature request] add docs about how to use OpenYurt to manage devices with EdgeX v2
**What would you like to be added**:
A tutorial of how to use OpenYurt to manage edge devices with EdgeX V2

**Why is this needed**:
yurt-device-controller has supported the new version EdgeX Foundry V2, so it's necessary to provide an updated tutorial.

**others**
/kind doc

### PR DESCRIPTION
**Pull request by wawlian**
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #34 

#### Special notes for your reviewer:
/assign @lwmqwer @gnunu 

#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

File: docs/yurt-device-controller-tutorial-v2.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+      actualValue: '[false, true, true, true, false]'
+      getURL: http://edgex-core-command:59882/api/v2/device/name/openyurt-created-random-boolean-device/BoolArray
+      name: BoolArray
+  edgeId: 01c87af0-1a0f-4227-b68d-4c03de19f25b
+  operatingState: UP
+  synced: true
+```
+
+The `deviceProperties` shows all the properties of this device. For example, the `Bool` property has a latest value `false`
+and the value this retrieved from the EdgeX rest api `http://edgex-core-command:59882/api/v2/device/name/openyurt-created-random-boolean-device/Bool`.
```
Comment (gnunu) [inline docs/yurt-device-controller-tutorial-v2.md:407]: this --> is

**Comment by lwmqwer** [comment]
I think we should put the tutorial of device-controller and edge manager together.  Give the use all-in-one doc to install, deployment, test rather than a separate docs in many places

**Comment by rambohe-ch** [comment]
> I think we should put the tutorial of device-controller and edge manager together. Give the use all-in-one doc to install, deployment, test rather than a separate docs in many places

@lwmqwer agree +1

File: docs/yurt-device-controller-tutorial-v2.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+```shell
+k patch device openyurt-created-random-boolean-device --type=merge -p '{"spec":{"managed":true,"deviceProperties":{"Bool": {"name":"Bool", "desiredValue":"true"}}}}'
+```
+
+In the command, we set the `Bool` DeviceProperty value to `true`, yurt-device-controller will trigger a EdgeX command
+and change the property of the device. We can check this by watch the status of device for multiple times, you will find
+the value is always `true` unless you change this property to `false` again.
+
+```shell
+watch "kubectl get device openyurt-created-random-boolean-device -o json | jq '.status.deviceProperties.Bool.actualValue'"
```
[the remark this answers is not in the document]
Comment (wawlian) [reply docs/yurt-device-controller-tutorial-v2.md:454]: Here, I just want to get actualValue of a property at regular intervals with the `watch` tool.

**Comment by wawlian** [comment]
@rambohe-ch @lwmqwer Provide an all-in-one tutorial is fine.  But maybe [github.io](https://github.com/openyurtio/openyurt.io) is a better place. This tutorial as an updated version puts more emphasis on how to manage edge devices  through yurt-device-controller from OpenYurt/K8S side.

File: docs/yurt-device-controller-tutorial-v2.md [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+
+### Create Device, DeviceService, DeviceProfile
+
+1. Create a DeviceService
+
+```yaml
+apiVersion: device.openyurt.io/v1alpha1
+kind: DeviceService
+metadata:
+  name: openyurt-created-deviceservuce-virtual
```
Comment (gnunu) [inline docs/yurt-device-controller-tutorial-v2.md:232]: deviceservuce -> deviceservice

**Comment by lwmqwer** [comment]
@wawlian correct, neither of device or edgex is suit for this tutorial .  But I mean is that we can just write a all-in-one and give the link in edgex and device is enough.

File: docs/yurt-device-controller-tutorial-v2.md [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+  managed: true
+  nodePool: hangzhou
+  notify: true
+  operatingState: UP
+  profileName: openyurt-created-random-boolean-deviceprofile
+  protocols:
+    other:
+      Address: openyurt-created-device-virtual-bool-01
+      Port: "300"
+  serviceName: openyurt-created-deviceservuce-virtual
```
[the remark this answers is not in the document]
Comment (wawlian) [reply docs/yurt-device-controller-tutorial-v2.md]: Done

**Review by qclc — COMMENTED** [review]
/approve

**Pull request merged**